solr/core/src/test/org/apache/solr/cloud/ZkCLITest.java

```diff
@@ -214,15 +213,11 @@ public void testPutFile() throws Exception {
 
     String fromZk =
         new String(zkClient.getData("/solr.xml", null, null, true), StandardCharsets.UTF_8);
-    File locFile = new File(SOLR_HOME + File.separator + "solr-stress-new.xml");
-    InputStream is = new FileInputStream(locFile);
-    String fromLoc;
-    try {
-      fromLoc = new String(IOUtils.toByteArray(is), StandardCharsets.UTF_8);
-    } finally {
-      IOUtils.closeQuietly(is);
+    Path locFile = Path.of(SOLR_HOME, "solr-stress-new.xml");
+    try (InputStream is = Files.newInputStream(locFile)) {
+      String fromLoc = new String(is.readAllBytes(), StandardCharsets.UTF_8);
+      assertEquals("Should get back what we put in ZK", fromZk, fromLoc);
     }
```

Comment:
Instead use `Files.readString(path)`

Reply:
Addressed in 7079308294cb7f4d4c468dbdc890afd2a6d67d2e